webapp/src/main/webapp/src/app/sender/archive/search-event.ts

```diff
@@ -0,0 +1,9 @@
+export class SearchEvent {
```

Comment:
I honestly see no use for this class, why do you need it?

Reply:
Done